app/models/session.rb

```diff
@@ -0,0 +1,50 @@
+class Session
+  include ActiveModel::Model
+  include Virtus.model
+  HOURS_TO_EXPIRE = 10
```

Comment:
👍 to environment variable, at least for TTL, and SessionStore can just be specified as a global variable in config/initializers/redis.rb which in tern references config/redis.yml

I had actually meant to do that in the first place, but this was quickly hacked together to demonstrate as there are far too many things going on and just want to get a very basic session and very basic controller endpoints setup for now. I'll make these changes though as I write specs.